src/overlays/actors/ovl_En_Holl/z_en_holl.c

```diff
@@ -171,9 +171,9 @@ void func_80A58DD4(EnHoll* this, GlobalContext* globalCtx) {
 
 // Horizontal Planes
 #ifdef NON_MATCHING
-// Small regalloc near the end
+// a/v regalloc near the end. Also ugly float thing in the conditional that seems needed to match.
 void func_80A59014(EnHoll* this, GlobalContext* globalCtx) {
-    s32 pad;
+    GlobalContext* pad;
```

Comment:
Can this not be s32?
```suggestion
    s32 pad;
```